src/MoneroEngine.ts

```diff
@@ -336,7 +342,8 @@ export class MoneroEngine implements EdgeCurrencyEngine {
       // Iterate over transactions in address
       for (let i = 0; i < transactions.length; i++) {
         const tx = transactions[i]
-        this.processMoneroTransaction(tx)
+        const blockHeight = this.processMoneroTransaction(tx)
+        this.highestBlockHeight = Math.max(this.highestBlockHeight, blockHeight)
```

Comment:
We don't know the order that transactions are returned from mymonero. If they are returned in reverse order then you'll be setting highestBlockHeight to the first transaction, then every transaction afterwards would have its blockHeight compared to it to determine `isNew` and they'd all be old. Safer to set a 

`const localBlockHeight = this.walletLocalData.blockHeight` at the top

Then at the end of checkTransactionsInnerLoop, set `this.highestBlockHeight = localBlockHeight`